content/neuvector-prime/azure/contents.lr

```diff
@@ -190,53 +199,46 @@ Billing will be available in Azure Portal billing
 Home > Cost Management <subscription> | Cost analysis
 
 
+## Enable Auto Upgrade minor version
 
-## Upgrading a NeuVector PAYG Cluster
+By default the Kubernetes Service Extension "Auto Upgrade minor version" will be in a Disabled state.
 
-The Azure marketplace PAYG offer is periodically updated as a new versions of NeuVector are released, and for optimizations in the integration with Azure. To update the latest version of the NeuVector PAYG offering supported in the marketplace listing, use the "az k8s-extension update" command.
+To enable Auto upgrade minor version
 
-As a Prerequisites to create extension instances in AKS cluster,  Register Azure Feature
+1. Click Extensions name
+2. Check Auto upgrade minor version 
 
-Execute the following command in Cluster Cloud Shell:
+This will take few seconds to update. In the Notifications, "Updating extension" job will be in running state. 
 
-```
-# Set your subscription.
-az account set --subscription <your-subId>
-
-# Register for preview features.
-az feature register --namespace Microsoft.KubernetesConfiguration --name extensionTypes
 
-# To check the state is registered
 
-az feature list -o table --query "[?contains(name, 'Microsoft.ContainerService/Microsoft.KubernetesConfiguration')].{Name:name,State:properties.state}"
+   !["Update progress"](./process.png)
 
-# Execute update command after the feature is registered.  
-az k8s-extension update  --name <cluster extension resource name>   --cluster-name <cluster name> --resource-group <resource group> --cluster-type managedClusters --version <version to be upgraded>
 
-# Update Example (Refer screenshot) 
-az k8s-extension update  --name testnv2-plan --cluster-name testnv2 --resource-group testnv-res-grp --cluster-type managedClusters --version 1.0.51
-
-```
+Once the "Update extension" job completed. Click "Refresh" in the Extension + application screen.
 
 
+   !["Enabled"](./enabled.png)
 
-In Azure Console, Cluster > Extensions + applications screen "Provisioning State" should be in "Updating" state.
 
 
 
 
-   !["Upgrade"](./upgradeinprogress.png)
 
 
+   ---
+   **NOTE**
 
+   You have an option to **temporarily enable** "Auto upgrade minor version" and then disable it after the upgrade completes.
```

Comment:
Maybe add "To manually control the timing of upgrade, "  before the "You have an option..."